topic_statistics_demo/README.md

```diff
@@ -0,0 +1,4 @@
+# Topic Statistics Demos
+
+The demo application in this package demonstrates topic statistics feature in ROS2.
```

Comment:
Here's a link to the [ros2 concepts page](https://index.ros.org/doc/ros2/Concepts/About-Topic-Statistics/) if you'd like to add it here.